### ISSUE TITLE
Different style for finish

### ISSUE THREAD

**Issue by hamzamohdzubair** [console-rs/indicatif#355]
Hi, I have been through the whole documentation but i am unable to find the solution for my usecase:
I want to start with a style:
```
let bar = indicatif::ProgressBar::new(10)
    .with_message("Converting")
    .with_style(indicatif::ProgressStyle::default_bar()
        .template("{msg:>15.cyan} {bar:40.cyan} {prefix} {percent:>3}%  [{elapsed}]"));
```

But I want to finish it with a different template where the bar is missing. I tried using `finish_using_style` but couldn't figure out how to make it work.

Thanks

**Comment by djc** [comment]
I think `finish_using_style()` is specifically about the `ProgressFinish` stored in the `ProgressStyle`. I don't think there is a way to do what you want today. If you want to contribute something, feel free to submit a PR, or maybe start with a brief description of the design you have in mind to avoid work that turns out to not fit in with my goals.

**Comment by hamzamohdzubair** [comment]
Yes i would love to contribute. I will try to briefly explain my proposal:

Add a method called `with_finish_style` to work something like this:

```rust
let bar = indicatif::ProgressBar::new(10)
    .with_message("Converting")
    .with_style(indicatif::ProgressStyle::default_bar()
        .template("{msg:>15.cyan} {bar:40.cyan} {prefix} {percent:>3}%  [{elapsed}]"));
    .with_finish_style(indicatif::ProgressStyle::default_bar()
        .template("{msg:>15.cyan} {prefix} {percent:>3}%  [{elapsed}]"));
```
In the above example, we have two styles: one before finish and one after finish, this allows me for example to remove the bar just after finish, but keep other information as is, I can submit a PR, but i just require some help getting started.

**Comment by djc** [comment]
What kind of help would you need? I think you'll find the code base reasonably straightforward to navigate.

**Comment by Yunuuuu** [comment]
Hope to add this feature

**Comment by eirnym** [comment]
Any news on that?

**Comment by luolong** [comment]
Sorry, I've been indisposed recently. Will check this out soon.

**Comment by eirnym** [comment]
You're welcome to edit or resubmit changes from my old PR, as the only placement wasn't agree with maintainers' of this crate.
